packages/synapse-react-client/src/synapse-queries/KeyFactory.ts

```diff
@@ -192,7 +193,12 @@ export class KeyFactory {
    * @private
    */
   private getKey(...args: any[]): QueryKey {
-    return [this.accessToken, ...removeTrailingUndefinedElements(args)]
+    return [
+      this.accessToken === undefined
+        ? this.accessToken
+        : btoa(String(hashCode(this.accessToken))),
+      ...removeTrailingUndefinedElements(args),
+    ]
```

Comment:
All react-query cache keys include the current access token. In the react-query devtools, the entire JWT is visible, which makes it hard to see the actual data and relevant key values. So, hash the token to shorten it and make the devtools usable.